cypress/integration/2-login/login.spec.ts

```diff
@@ -54,4 +54,39 @@ describe("Login", () => {
             cy.url().should('contain', '/#/home');
         });
     });
+
+    describe("logout", () => {
+        beforeEach(() => {
+            cy.initTestUser(synapse, "Erin");
```

Comment:
It seems like we use a different displayname every time we do `initTestUser`. Is there a specific purpose? Does this make something more clear somehow?

Reply:
No real purpose, the old test suite used the name Erin and the tests were ported from there.